<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentitySettingsWidget</class>
 <widget class="QWidget" name="IdentitySettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Identity</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="qlNickname">
       <property name="text">
        <string>Nic&amp;kname:</string>
       </property>
       <property name="buddy">
        <cstring>qleNickname</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="qleNickname"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="qlUsername">
       <property name="text">
        <string>Userna&amp;me:</string>
       </property>
       <property name="buddy">
        <cstring>qleUsername</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="qlRealname">
       <property name="text">
        <string>&amp;Real name:</string>
       </property>
       <property name="buddy">
        <cstring>qleRealname</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="qcbInvisible">
       <property name="text">
        <string>&amp;Invisible mode (+i)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="qcbWallops">
       <property name="text">
        <string>&amp;Wallops (+w)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="qleUsername"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="qleRealname"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="qlQuitMessage">
       <property name="text">
        <string>&amp;Quit message:</string>
       </property>
       <property name="buddy">
        <cstring>qleQuitMessage</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="qleQuitMessage"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qleNickname</tabstop>
  <tabstop>qleUsername</tabstop>
  <tabstop>qleRealname</tabstop>
  <tabstop>qleQuitMessage</tabstop>
  <tabstop>qcbInvisible</tabstop>
  <tabstop>qcbWallops</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
